docs/how-we-work.md

```diff
@@ -1,97 +1,100 @@
-
-2.  **Understand what people need.** We practice human-centered design to inform technical and design decisions. We continually explore and pinpoint the needs of the people who use the service and build functionality to meet their needs. We design an intuitive system that allows users to succeed the first time, unaided, and follow accessibility best practices to ensure all people can use the service.
-
-3.  **Own and manage our data.** We ensure that all Court data remains in the Court's care, custody, and control.
+## Tools
 
-4.  **Own and manage our software.** We retain control over our mission-critical custom software, ensuring portability and flexibility by using service-oriented architecture.
+-   GitHub: We use our [GitHub organization](https://github.com/ustaxcourt/ef-cms) for storing both software and collaboratively-maintained text.
+-   ZenHub: We use the vendor's [ZenHub organization](https://github.com/flexion/ef-cms#workspaces/ef-cms-5bbe4bed4b5806bc2bec65d3/boards?repos=152320868&showPipelineDescriptions=false) for storing the product backlog.
+-   Zoom: We use both the Court's and vendor's Zoom accounts for video teleconferences.
+-   Slack: We use both the Court's and vendor's Slack for communication that falls outside of the structure of GitHub or ZenHub, but that doesn't rise to the level of email, or for communication that it's helpful for everybody else to be able to observe. Notable channels include:
+    - #ustcproject — Main project channel. Used to communicate high-level messages re: project
+    - #ustc-story-channel — Channel used to communicate story-related work.
+    - #ustcux — Design/research-specific channel. 
+    - #ustctech — Tech-specific channel.
 
-5.  **Default to open.** We build secure, open source software, creating and maintaining it in public repositories, where it may be inspected by the public who places their trust in it, and where it is available for use by other courts.
+## Meetings
 
-6.  **Build atop a modern technology stack.** We use cloud-based hosting infrastructure, modular software frameworks, containerization, and automated testing to increase efficiency, allow the team to work effectively, and scale the service on demand.
+There are two basic meeting rhythms: daily design calls/standups and bi-weekly agile sprint rituals. All meetings are held via video teleconference. A telephone bridge is maintained as a backup method of connecting, but participants are strongly encouraged to join via video.
 
-7.  **Favor emergent architecture over "big up-front design."** Consistent with agile, we optimize for serving the emergent needs of users, instead of planning and executing an architecture that cannot respond to those changing needs.
+### Daily
 
-8.  **Deliver incremental value.** We deliver value to users as often as possible.
+- **Daily design sync** — this meeting is a time to solution functionality, demo in-progress work with the team and review new wireframes
+- **Daily standup**
 
```

Comment:
Might want to add something about our Monday & Friday DAWSON touch-base with Judge Buch and Stephanie.

Reply:
... a stake holder meeting

### Weekly

- **Stakeholder Meeting**  - on Mondays and Fridays, we review the current status with stakeholders at the Court and discuss the schedule of large releases that need off-hours deployments.